gradle.properties

```diff
@@ -11,5 +11,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-version=1.1.2
+version=2.0.0-SNAPSHOT
```

Comment:
Isn't this the version of rsocket-java? :)

Reply:
Yeah I thought that this might be released as RSocket 2.0.0 cause we're introducing micrometer that introduces breaking changes. We'll see if it actually remains 2.0.0 or 1.2